client/src/schemas/material.js

```diff
@@ -67,56 +78,8 @@ const importedCategories = {
     number_of_available_models: string(),
     construct_details: string()
   }),
-  // this currently can't be imported
-  // when it can remove keys which have no inputs
-  // mark required when applicable
-  PDX: object({
-    patient_id: string(),
-    gender: string(),
-    age: string(),
-    diagnosis: string(),
-    consent_to_share: string(),
-    ethnicity: string(),
-    treatment_drug: string(),
-    treatment_protocol: string(),
-    prior_treatment_protocol: string(),
-    response_to_treatment: string(),
-    virology_status: string(),
-    submitter_tumor_id: string(),
-    tissue_of_origin: string(),
-    tumor_type: string(),
-    specimen_tumor_tissue: string(),
-    histology: string(),
-    tumor_grade: string(),
-    disease_stage: string(),
-    specific_markers: string(),
-    is_from_untreated_patient: boolean(),
-    tumor_sample_type: string(),
-    existing_model_explanation: string(),
-    pdx_id: string(),
-    model_organism: string(),
-    model_strain_and_source: string(),
-    is_strain_immunized: boolean(),
-    type_of_humanization: string(),
-    tumor_preparation: string(),
-    injection_type_and_site: string(),
-    treatment_for_engraftment: string(),
-    engraftment_rate: string(),
-    engraftment_time: string(),
-    tumor_characterization_technology: string(),
-    is_not_of_ebv_origin: string(),
-    is_passage_qa_performed: boolean(),
-    response_to_qa_performed: string(),
-    response_to_standard_of_care: string(),
-    treatment_passage: string(),
-    treatment_response: string(),
-    tumor_omics: string(),
-    metastases_in_strain: string(),
-    lag_time: string(),
-    number_of_available_models: string(),
-    pdx_model_availability: string(),
-    governance_restriction: string()
-  }),
+  // PDXs currently can't be imported
+  PDX: object(),
   OTHER: object({
```

Comment:
Is this intentional? It looks like PDX is an empty object and OTHER is defined twice.

Reply:
1) Yeah it just means additional attributes should always be an object but doesnt validate for keys. This is in case import is toggled. It wont validate but I just kept the prop there. 
2) it just looks like that from the code folding. The `OTHER:` here is for imported materials the next one is for listed.

Reply:
Oops that was just cold folding. Good call :+1: